app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoiceFullShipmentGroupedProductCustomStock.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminCreateCreditMemoTotalRefundFullInvoiceFullShipmentGroupedProductCustomStock" extends="AdminCreateCreditMemoTotalRefundFullInvoiceFullShipmentGroupedProductDefaultStock">
+        <annotations>
+            <stories value="Grouped Product Custom Stock."/>
+            <title value="Create Credit Memo with Total Refund after Full Invoice and Full Shipment with Grouped product on Custom stock."/>
+            <description value="Verify, admin able to create credit memo with total refund after full invoice and full shipment with grouped product on custom stock."/>
+            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/2553502"/>
```

Comment:
This is already linked to `AdminCreateCreditMemoTotalRefundGroupedProductCustomStock`

Please investigate and fix

Reply:
Same as above, both tests have different links
Grouped with simple product - https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/2553502
Grouped with virtual product - https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/1620169